services/app-web/src/constants/routes.ts

```diff
@@ -101,8 +101,8 @@ const QUESTION_RESPONSE_SHOW_SIDEBAR_ROUTES: RouteTWithUnknown[] = [
 */
 const PageHeadingsRecord: Partial<Record<RouteTWithUnknown, string>> = {
     ROOT: 'Dashboard',
-    DASHBOARD_SUBMISSIONS: 'Dashboard',
-    DASHBOARD_RATES: 'Dashboard',
+    DASHBOARD_SUBMISSIONS: 'Submissions Dashboard',
```

Comment:
You'll have to update the cypress tests for this change. We use this header to as a way to wait for the page to load.